Docs/XCHammerFAQ.md

```diff
@@ -81,6 +81,56 @@ populated after the first Bazel build.._
 
 ## Development
 
+### How can I develop and debug XCHammer?
+
+The `Makefile` contains commands for development and distribution. `make run` and
+`make run_force_bazel` both run the sample app with a debug build.
+
+*The sample is a great way to do development on, and the authors recommend
+adding examples of edge cases and bugs here :). Pull requests welcome!*
+
+
+### XCHammer local repository integration
+
+When pulling in XCHammer with `git_repository` and building it in tree, simply
+override it with a local repository in the iOS repo's build or `.bazelrc`
+
+```
+build --override_repository=xchammer=/path/to/xchammer/
+```
+
+### XCHammer binary WORKSPACE integration for development
+
+For binary integration / out of tree builds, `make xchammer_dev_repo` allows a
+`WORKSPACE` to consume XCHammer built out of tree but point to the latest build,
+symlinked in `bazel-bin`. Add to the repository in question:
+```
+build --override_repository=xchammer=/path/to/xchammer/bazel-bin/xchammer_dev_repo
+```
+
+This makes it easy and pain free to build XCHammer outside of the repo but pull
+it into the `WORKSPACE`
+
+### Debugging XCHammer Bazel rules in Xcode
+
+1. It the iOS repository, override `xchammer` in the .bazelrc to point to the
```

Comment:
```suggestion
1. In the iOS repository, override `xchammer` in the `.bazelrc` to point to the
```

Reply:
@jparise good catch and also thank you kindly for the fix 🙏